recipes/spix/all/test_package/conanfile.py

```diff
@@ -17,12 +17,7 @@ def requirements(self):
     def layout(self):
         cmake_layout(self)
 
-    def _patch_sources(self):
```

Comment:
Linter fflags the impoorts here i nthis file

Reply:
Fixed

Reply:
Wow my typing was bad lol